config/zones/IN-SO.yaml

```diff
@@ -18,6 +18,23 @@ contributors:
   - gopikrishna1793
   - unitrium
 timezone: Asia/Kolkata
+emissionFactors:
+  direct:
+    unknown:
+      - datetime: '2020-01-01'
+        source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
```

Comment:
@unitrium do you have a URL reference for a Central Electricity Authority page / document where the Y2022 report can be found?

If possible I'd like to add it while attempting to move as many of the `source` references as I can into top-level zone `sources` entries as part of #6001 (generally both a source name and a `link` attribute for it seem like best-practice).